src/mame/drivers/tetrisp2.cpp

```diff
@@ -1860,53 +1839,174 @@ TIMER_DEVICE_CALLBACK_MEMBER(stepstag_state::field_cb)
 	// irq 4 is definitely a 30 Hz-ish here as well,
 	// except we have a multi-screen arrangement setup and no way to pinpoint source
 	m_subcpu->set_input_line(4, HOLD_LINE);
+	stepstag_state::simulate_pc();	// any better place to put it into schedule?
 }
 
 void stepstag_state::setup_non_sysctrl_screen(machine_config &config, screen_device *screen, const XTAL xtal)
 {
 	// TODO: unknown clock and parameters
-	// assume there's a 42.954 MHz/6 like nndmseal to compensate the higher res
-	screen->set_raw(xtal/6, 455, 0, 352, 262, 0, 240);
+	screen->set_raw(xtal, 429, 0, 352, 525, 0, 240);	// should be 0-479 interlaced
```

Comment:
If it's supposed to superimpose for the FMVs then this change is structurally erratic 
i.e. there are two sources that needs to be post-mixed, as it can't possibly run on the same CRTC clock.

Also the inline note belongs to the TODO itself, how do you know it's 525i anyway?

Reply:
Only ONE source needs to be post-mixed with the FMV: the in-game video which is from the sub-cpu-board.
Self-Test Text signal is output from the main-cpu-board. It doesn't need mixing.

525i(or 262p/263p) is verified by some video footages taken by standard ntsc video camcord, by observing the scanning rate Interference fringes(horizental dark bar rolling).
Some youtube footage of both VJ and Stepping Stage also help,  For example:
https://www.youtube.com/watch?v=bjrhNC54h0o
In this video of VJ, the scanning rate of the 3 upper screens is different from the lower(main) one. The 3 upper screens(output from the sub-cpu-board) are in ntsc-like rate, while the lower main game screen is flickering more, as it connect to a tetrisp2/rockn board(the main cpu board).

Reply:
imho it just sounds as speculation, if you don't have actual tested proof my guess is as good as yours.

Reply:
One evidence is: 
The palette of the sub-cpu's video hardware is in YUV format, usually used in FMV;
While, the palette of the main-cpu's video is in RGB.